src/payment/models/history/dto/history.dto.ts

```diff
@@ -21,3 +29,7 @@ export class HistoryDto extends HistoryDtoBase {
 export class CoinTrackingHistoryDto extends HistoryDtoBase {
   date: number;
 }
+
+export class SimplifiedHistoryDto extends RouteHistoryDto {
```

Comment:
It might be a good moment to also swagger-document the route history return DTOs 😉